backend/core/app_initializer_v2.py

```
@@ -100,13 +101,21 @@ def initialize_app_state() -> Tuple[Dict[str, Any], SmartIllustrationService, Ba
-        logger.info(message)
+        logger.info(f"✅ {message}")
 
     # Store unified retriever for direct access if needed
     all_retrievers["_unified_retriever"] = unified_retriever  # type: ignore[assignment]
 
+    # Warmup settings cache during app initialization
+    try:
+        settings_manager = get_settings_manager()
+        settings_manager.warmup_cache()
+        logger.info("✅ Settings cache warmed up during app initialization")
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)